jobs/cloud_controller_worker/spec

```diff
@@ -425,6 +426,21 @@ properties:
     description: "The number of seconds to wait for each generic cloud_controller_worker worker process to finish processing jobs before forcefully shutting it down"
     default: 15
 
+  cc.publish_metrics:
+    default: false
+    description: "When set to true a small webserver will be started in a seperate thread within the first worker's process.
+                  This webserver will publish prometheus metrics of the workers under '/metrics'. The webserver will listen on the port
+                  defined in 'cc.prometheus_port'."
+  cc.prometheus_port:
+    default: 9394
+    description: "When 'cc.publish_metrics' is set to true, the webserver, which publishes the metrics, will listen on this port."
+  cc.prom_scraper.disabled:
+    default: false
+    description: "When 'cc.publish_metrics' is enabled, a prom_scraper_config will be automatically generated. If you want to use another component for scraping, you can disable scraping by prom_scraper for cc-worker metrics with this."
+
+  cc.directories.tmpdir:
```

Comment:
Does this need to be configurable? Other jobs just use a hard-coded "tmp" path in their own "data" folder.

Reply:
Not necessarily, but `cloud_controller_ng` job does it in the same way. I like this more than hardcoding the path in the code of the rake task.